src/background/handlers/fetchAndStoreHistory.js

```diff
@@ -0,0 +1,75 @@
+import pLimit from 'p-limit';
+import { db } from '../initdb.js';
+import { shouldBlockDomain } from '../services/blacklist.js';
+import {
+  storeHistoryItems,
+} from '../services/db.js';
+import { ensureEngineIsReady, classifyURLAndTitle } from '../services/ml.js';
+
+export async function fetchAndStoreHistory(days) {
+  const now = Date.now();
+  const startTime = now - days * 86400e3;
```

Comment:
what is this `86400e3` ? let's use a CONSTANT_WITH_EXPLICIT_NAME so we understand what it is